outlines/text/prompts.py

```diff
@@ -284,7 +284,7 @@ def get_schema_dict(model: Dict):
 
 
 @get_schema.register(type(BaseModel))
-def get_schema_pydantic(model: type[BaseModel]):
+def get_schema_pydantic(model: type(BaseModel)):
```

Comment:
@mlelarge, mind rebasing and removing this one?  If not, no worries; we can do it from here.

Reply:
Not sure to understand what you want me to do. Probably better if you take it over from here. Sorry...